tests/unit/mathematics/linalg/LinalgBackendViennaCL_unittest.cc

```diff
@@ -5,7 +5,7 @@
 #include <thread>
 
 #ifdef HAVE_VIENNACL
-#include <shogun/mathematics/linalg/LinalgBackendViennaCL.h>
+#include <shogun/mathematics/linalg/LinalgBackendViennacl.h>
```

Comment:
@vigsterkr Can I just fix it quickly by pushing it directly to develop branch? Or you want to wait till when I look into other problems in ViennaCL? Or @lambday just fix it here?